src/core.js

```diff
@@ -497,15 +490,16 @@ goog.require('olcs.core.OlLayerPrimitive');
 
 
   /**
-   * Convert an OpenLayers geometry to 4326 projection.
-   * The geometry will be cloned only when reprojection is required.
-   * The properties will be shallow copied.
+   * Reproject an OpenLayers geometry to EPSG:4326 if needed.
+   * The geometry will be cloned only when original projection is not EPSG:4326
+   * and the properties will be shallow copied.
    * @param {!T} geometry
    * @param {!ol.proj.ProjectionLike} projection
    * @return {!T}
    * @template T
+   * @api
    */
-  var olGeometryCloneTo4326 = function(geometry, projection) {
```

Comment:
You have sharp eyes @elemoine. It is indeed a good opportunity to remove it and it will
be valuable when testing. I just commited this internal change.